### ISSUE TITLE
Thumbnails on the Publication Cover page aren't accessible

### ISSUE THREAD

**Issue by justinepocock** [localgovdrupal/localgov_publications#234]
Sarah Quinn (Blackburn with Darwen BC) pointed out that we use Silktide to monitor the accessibility on our site and are getting the a warning about the thumbnails on publication cover pages to 'Ensure links contain text'. Has anyone else had this and is there anything we need to do to fix this? Do the thumbnails need to be a link?

I believe in initial user testing people did click the thumbnail icon so might be best to add appropriate text here, until we know more.

**Comment by andybroomfield** [comment]
Is there alt text on the icon? And would it be different for each publication link?

**Comment by markconroy** [comment]
I'm checking this on a default installation of LGD + Publications and not getting any errors reported. When I checked the code, we have the publications icons as SVG, but they are not linked and they also have the `aria-hidden="true"` attribute so should not have a need for alt text.

![Image](https://github.com/user-attachments/assets/fdb04830-76e1-4576-903f-b70072a0c2a7)

If I can get a link to a publication on the B&D site, I can verify if it's particular to their setup/installation.

**Comment by markconroy** [comment]
Looks like on the B&D theme they have wrapped the SVG in a `<a>` tag, so that's what's causing the issue.

![Image](https://github.com/user-attachments/assets/4b07c690-2511-4854-a9e0-47b7b986d3eb)

**Comment by Swarbs1991** [comment]
Hi all, thanks for checking this out for us.

I've had a quick look just now and there's nothing in our subtheme to be overriding this so I believe the template file loading this is the media--document--publication.html.twig from the publications module, but just the bottom PDF file - looks like the top HTML one is defaulting to field-group-html-element.html.twig indicated by the green tick on Twig dump on our dev site.

I can override the module's template in the subtheme and just fix our issue by taking out the anchor tag, but looking at the LGD Demo site the SVGs are wrapped in `<a>` tags like our site which conflicts with what @markconroy is seeing on his fresh install? Bit confused on multiple points here, but nothing new there...

**Comment by markconroy** [comment]
Thanks Joe, I'll have a check of this again. Hopefully tomorrow.

**Comment by Swarbs1991** [comment]
Just been looking at the Publications templates to see if I can save anyone some time investigating this and the templates do wrap the SVGs in `<a>` tags. Don't suppose the fix would be just to propose removing these from the templates, just around the icons at least?

I haven't got any way of checking to see if this would sate SilkTide as we can only check our live site. Had a play with the Sa11y module but that doesn't flag it as an issue.

**Comment by markconroy** [comment]
I'm testing this again with latest version of publications and I can see that the SVG _is wrapped_ in an `<a>` tag, but the `<a>` tag has `aria-hidden="true"` set it should be ignored by screenreaders. 

Testing it on VoiceOver on Mac, and it's being ignored properly. Can anyone confirm via other screenreaders that the SVG link is being correctly ignored? If so, then this might be a fault in SilkTide's reporting rather than a bug for us to fix.

**Comment by Swarbs1991** [comment]
You're right Mark I've just tested this in a couple of browsers with screen readers and also ReachDeck and seems to skip over the SVGs and links as expected. Guess this is a red herring on our SilkTide dashboard.

**Comment by markconroy** [comment]
Thanks Joe, I'll close this issue so. We can re-open it if we need to later.

**Comment by Swarbs1991** [comment]
Thanks Mark, really appreciate you looking at this.
